doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -49,7 +49,7 @@ create_cluster ::=
     'REPLICAS' '(' (replica_definition (',' replica_definition)*)? ')'
   )?
 create_managed_cluster ::=
-  'CREATE' 'CLUSTER' name '(' 'MANAGED' (',' cluster_option '=' value)* ')'
+  'CREATE' 'CLUSTER' name 'SIZE' '=' value ',' 'REPLICATION' 'FACTOR' '=' value (',' cluster_option '=' value)*
```

Comment:
Isn't `REPLICATION FACTOR` optional? As in, if not specified, it defaults to `1`.

Reply:
```sql
materialize=> CREATE CLUSTER t SIZE = '3xsmall';
CREATE CLUSTER

materialize=> SELECT * FROM mz_clusters WHERE name = 't';
  id  | name | owner_id |     privileges     | managed |  size   | replication_factor | disk
------+------+----------+--------------------+---------+---------+--------------------+------
 u258 | t    | u2       | {s2=U/u2,u2=UC/u2} | t       | 3xsmall | 1                  | f
(1 row)
```

Reply:
Yup, updated.